core/README.md

```diff
@@ -23,9 +23,7 @@ the [`deno_ops`](https://github.com/denoland/deno/blob/main/ops) crate.
 
 Documentation for this crate is thin at the moment. Please see
 [hello_world.rs](https://github.com/denoland/deno/blob/main/core/examples/hello_world.rs)
-and
-[http_bench_json_ops.rs](https://github.com/denoland/deno/blob/main/core/examples/http_bench_json_ops.rs)
```

Comment:
Can you link to `https://github.com/denoland/deno/blob/main/core/examples/http_bench_json_ops/main.rs` instead?